### PR TITLE
Commands reworked to allow for permission levels

### PR DESCRIPTION
**Pull request by SaschaMann**

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -2,4 +2,5 @@ twisted
 requests
 riotwatcher
 redis
```
Comment (RonakR) [inline requirements.txt]: I think we can get rid of this now

File: triviabot/plugins/scores.py [hunk 2]
```diff
@@ -1,13 +1,13 @@
 from functools import partial
 
-from triviabot.models import User
+from triviabot.models import User, Command, Event
```
Comment (RonakR) [inline triviabot/plugins/scores.py]: `Event` is an unused import.

File: triviabot/plugins/help.py [hunk 3]
```diff
@@ -1,32 +1,24 @@
...
+    bot.add_command(Command('help', helpme, 'Displays a list of available commands.'))
 
 
 def helpme(bot, user, channel, args):
     message = 'The following commands are available: '
 
-    # Generates a list of available commands
-    for command in command_dict:
-        message += command + ' '
+    message += ', '.join([command.trigger for command in bot.commands.itervalues()])[:-2]
```
Comment (FichteFoll) [inline triviabot/plugins/help.py]: A list comprehension inside a join call is entirely redundant, btw, and you can instead just leave the generator expression.

Also, why the `[:-2]` slice at the end? Join only adds `', '` as a separator. Looks like a bug.

File: triviabot/plugins/help.py [hunk 4]
```diff
@@ -9,7 +9,7 @@ def on_load(bot):
 def helpme(bot, user, channel, args):
     message = 'The following commands are available: '
 
-    message += ', '.join([command.trigger for command in bot.commands.itervalues()])[:-2]
+    message += ', '.join(command.trigger for command in bot.commands.itervalues()) + '.'
```
Comment (FichteFoll) [inline triviabot/plugins/help.py]: I personally wouldn't add a dot here, but not a blocker.

**Pull request merged**